bundles/org.openhab.binding.sensibo/src/main/java/org/openhab/binding/sensibo/internal/handler/SensiboAccountHandler.java

```diff
@@ -0,0 +1,328 @@
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.TypeAdapter;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+/**
+ * The {@link SensiboAccountHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Arne Seime - Initial contribution
+ */
+@NonNullByDefault
+public class SensiboAccountHandler extends BaseBridgeHandler {
+    private static final int MIN_TIME_BETWEEEN_MODEL_UPDATES_MS = 30_000;
+    private static final int SECONDS_IN_MINUTE = 60;
+    public static String API_ENDPOINT = "https://home.sensibo.com/api";
+    private final Logger logger = LoggerFactory.getLogger(SensiboAccountHandler.class);
+    private final HttpClient httpClient;
+    private RequestLogger requestLogger;
+    private Gson gson;
+    private SensiboModel model = new SensiboModel(0);
+    private @Nullable ScheduledFuture<?> statusFuture;
+    private @NonNullByDefault({}) SensiboAccountConfiguration config;
+    private final JsonParser jsonParser = new JsonParser();
+
+    public SensiboAccountHandler(final Bridge bridge, final HttpClient httpClient) {
+        super(bridge);
+        this.httpClient = httpClient;
+
+        gson = new GsonBuilder().registerTypeAdapter(ZonedDateTime.class, new TypeAdapter<ZonedDateTime>() {
+            @Override
+            public void write(final @NonNullByDefault({}) JsonWriter out, final ZonedDateTime value)
+                    throws IOException {
+                out.value(value.toString());
+            }
+
+            @Override
+            public ZonedDateTime read(final @NonNullByDefault({}) JsonReader in) throws IOException {
+                return ZonedDateTime.parse(in.nextString());
+            }
+        }).setLenient().setPrettyPrinting().create();
+
+        requestLogger = new RequestLogger(bridge.getUID().getId(), gson);
+    }
+
+    private boolean allowModelUpdate() {
+        final long diffMsSinceLastUpdate = System.currentTimeMillis() - model.getLastUpdated();
+        return diffMsSinceLastUpdate > MIN_TIME_BETWEEEN_MODEL_UPDATES_MS;
+    }
+
+    public SensiboModel getModel() {
+        return model;
+    }
+
+    @Override
+    public void handleCommand(final ChannelUID channelUID, final Command command) {
+        // Ignore commands as none are supported
+    }
+
+    public SensiboAccountConfiguration loadConfigSafely() throws SensiboConfigurationException {
+        SensiboAccountConfiguration loadedConfig = getConfigAs(SensiboAccountConfiguration.class);
+        if (loadedConfig == null) {
+            throw new SensiboConfigurationException("Could not load Sensibo account configuration");
+        }
+
+        return loadedConfig;
+    }
+
+    @Override
+    public void initialize() {
+        scheduler.execute(() -> {
```

Comment:
It is a good practice to make sure that the thing status is set to something before initialize() returns.
```suggestion
    public void initialize() {
        updateStatus(ThingStatus.UNKOWN);
        scheduler.execute(() -> {
```

Reply:
Fixed